dashboards-observability/public/components/visualizations/charts/bar/bar.tsx

```diff
@@ -82,12 +78,10 @@ export const Bar = ({ visualizations, layout, config }: any) => {
     ...layout,
     ...(layoutConfig.layout && layoutConfig.layout),
     title: dataConfig?.panelOptions?.title || layoutConfig.layout?.title || '',
-    barmode:
-      dataConfig?.chartOptions?.mode &&
-      dataConfig.chartOptions.mode[0] &&
-      dataConfig.chartOptions.mode[0].modeId
-        ? dataConfig.chartOptions.mode[0].modeId
-        : '',
+    barmode: dataConfig?.chartStyles?.mode || visualizations.vis.mode,
```

Comment:
Is bwc supported here after this change? If the user has a previously saved visualization with chartOptions, would it break?